sdk/js/src/Client.ts

```diff
@@ -114,7 +117,8 @@ export class DVCClient implements Client {
         if (this.config) {
             config = this.config
         } else {
-            throw new Error('Client not initialized')
+            this.userQueue.queueIdentify(user, callback)
```

Comment:
what happens if the client is already initialized and you call "identify" multiple times. Won't that cause parallel config requests to happen? Should we queue those as well?